### PR TITLE
Chapters 4, 5, 6

### PR DESCRIPTION
**Pull request by pykong**
**ChangeLog:**
- Added examples to chapter 4
- Added chapter 5 - "The original manuscript
- Add chapter 6 - plagiarism (Noite there is more content in Unit 6 of the course which I did not include because I am not interested in that.)
- Added section on Further Reading

**TODO:**
- Section numbering and TOC not updated (@SebastianElvis do you got some programmatic way to do this?)

**Note:**
@SebastianElvis I have collected all changes into a single big changeset out of convenience. If you like to have smaller changesets I need to split it up into smaller chapter-focus PRs.

### CODE REVIEW

**Comment by SebastianElvis** [comment]
Thanks for your continuous contribution!

> * Section numbering and TOC not updated (@SebastianElvis do you got some programmatic way to do this?)

I use VSCode's Markdown plugin, which supports generating ToC automatically. I believe it's based on this project https://github.com/jonschlinkert/markdown-toc. You can run it as a standalone program as well.

> Note:
> @SebastianElvis I have collected all changes into a single big changeset out of convenience. If you like to have smaller changesets I need to split it up into smaller chapter-focus PRs.

It (including everything in a single PR) is okay to me.

**Comment by pykong** [comment]
@SebastianElvis I have updated section numbering and TOC.
Doing this manually is acceptable for a finite scope project such as this, but for more elaborate cases (e.g. doctoral theses) `pandoc` is better suited to take care of this task.

Anyway thanks for laying the groundwork of this project.

I am not going to include the other units as they are not focused on writing papers and theses.

**Comment by pykong** [comment]
@SebastianElvis is there anything missing to get this PR merged?

File: README.md [hunk 1]
```diff
@@ -1164,6 +1214,665 @@ The reader may think you are referring to a different instrument, model, group,
...
+
+##### 4.6.2 EXAMPLE 2
+
+Bad:
+
+```
+The church’s record-keeping system and its high level of accuracy has been previously reported (2). Briefly, the church creates and maintains a church record for each individual who is baptized into the church. These records are created at the congregation level at the time of baptism and then forwarded to the general church level where the membership record is added to the church membership database. These records contain minimal information including name, date of birth,
+parents’ names, dates of church ordinances, and current address. Each congregation has a lay membership clerk who is responsible for updating the membership records for the members of the congregation. Such updates would include dates of ordinances received after baptism (e.g., priesthood ordination or marriage), name of spouse when a member marries, change of current address and date of death.
+```
+- [(WIP) Academic Writing Style Guide for Programmers](#wip-academic-writing-style-guide-for-programmers)
```
Comment (SebastianElvis) [inline README.md]: Why is the ToC there?

**Review by SebastianElvis — CHANGES_REQUESTED** [review]
Other parts are good to me

File: README.md (hunk 1 above)
Comment (pykong) [reply README.md]: Ouch. I missed that. It is fixed now.

**Comment by SebastianElvis** [comment]
Thanks!

**Pull request merged**